rbac/server/api/swagger.py

```diff
@@ -16,11 +16,17 @@
 from sanic import Blueprint
 from sanic import response
 
+from rbac.common.logs import get_default_logger
+
+LOGGER = get_default_logger(__name__)
+
 SWAGGER_BP = Blueprint("swagger")
 
 
 @SWAGGER_BP.get("api/swagger")
 async def get_swagger(request):
+    if not request:
+        LOGGER.debug(str(request))
```

Comment:
Does removing request not work here?

Reply:
It does not because request is still not being used anywhere in the method

Reply:
right so can you not remove request from the parameters?  Or is it required?

Reply:
It is required, unfortunately.